dev.zed.Zed.Preview.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.zed.Zed.Preview</id>
```

Comment:
this should be `dev.zed.ZedPreview` instead. But there is something about how flathub is for released software....

Reply:
should I also make this the `app-id`?

> But there is something about how flathub is for released software....

Sure, but zed (as most other rust projects), moves too quickly. some features are released to bake in here. Zed stable itself is also alpha so...

Reply:
> should I also make this the app-id?

What do you think?

Reply:
I think so, yes. If that's what's recommended

Reply:
Needs to rename to dev.zed.Zed-Preview to match with Zed ID.
https://github.com/zed-industries/zed/blob/e07ffe7cf180869fb6cf90700d7bf12ae3c8ee95/crates/release_channel/src/lib.rs#L158

Reply:
thanks